src/main/java/com/kainos/discoverydiary/models/DiaryEntry.java

```diff
@@ -8,20 +8,25 @@
  * DiaryEntry object
  */
 public class DiaryEntry {
+
+    final String DEFAULT_IMAGE_URL="http://i.imgur.com/BbpaP7N.png";
     private SessionType sessionType;
     private String title;
     private String startDate;
     private String startTime;
     private String sessionGoal;
+    private String imageUrl;
     private int projectID;
 
+
     public DiaryEntry(SessionType sessionType, String title, String startDate, String startTime, String sessionGoal,
```

Comment:
Could you create an overloaded method so that we do not have to add in the default image url as "", instead we would just call the method without the parameter